TypeApprovalRegister+data.json

```diff
@@ -1755,12 +1625,12 @@
                   {
                     "local-id": "000",
                     "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_MANAGEMENT",
-                    "logical-termination-point": "tar-1-0-0-op-s-0000"
+                    "logical-termination-point": "tar-1-0-0-op-s-bm-000"
```

Comment:
The file looks good for converting uuid(s) from service.yaml file to data.json file. 
Additionally, we need to update uuid(s) of forwarding-construct.
For example, in line 1610, uuid of fc tar-1-0-0-op-fc-0000 to be updated to "tar-1-0-0-op-fc-bm-000". This can be referred from [TypeApprovalRegister+forwardings.yaml](https://github.com/openBackhaul/TypeApprovalRegister/blob/develop/TypeApprovalRegister%2Bforwardings.yaml) file. From this file, we can search for the corresponding ForwardingName , (in this case, ForwardingName=PromptForRegisteringCausesRegistrationRequest) and copy it's corresponding fc-uuid and paste it in data file for specified location